changelog/unreleased/kong/prometheus-upstream-metrics-toggle.yml

```diff
@@ -0,0 +1,3 @@
+message: "**Prometheus**: Use the :configure() handler to toggle upstream_health_metrics"
```

Comment:
nit: `refactor` would be more accurate (unless we can elaborate a bit more on the bug it fixes)

Reply:
agree, but I guess `refactor` isn't a valid type according to the changelog tool :(

Reply:
My bad. I meant a “chore”.